### ISSUE TITLE
Definition of Loss

### ISSUE THREAD

**Issue by pidahbus** [google-research/electra#15]
While training I find there is one loss. Is it training loss or validation loss?

**Comment by clarkkev** [comment]
It is training loss. You can run with `"do_train": false, "do_eval": true` to get the loss in eval mode (no dropout), but we did not use a separate validation set because the corpus is large enough that the model does not overfit.

**Comment by pidahbus** [comment]
Could you please suggest the proper parameter settings if I want to pre-train on a smaller dataset?
